utility/build-homebrew.sh

```diff
@@ -48,7 +48,7 @@ $(cat "$REQS_FILE")
 REQS
 
     File.write("requirements.txt", reqs)
-    system "python3", "-m", "pip", "install", "-r", "requirements.txt", "--require-hashes", "--no-deps", "--ignore-installed", "--prefix", libexec
+    system "python3", "-m", "pip", "install", "-r", "requirements.txt", "--no-deps", "--ignore-installed", "--prefix", libexec
```

Comment:
@Br4nd0R I don't think we want to remove it here; the homebrew installer is for Macs so we should be able to rely on it including the correct dependencies for the platform as long as we're building the homebrew formula from a Mac.

Reply:
@Br4nd0R Why is this marked resolved?

Reply:
Sorry for resolving that thread. I was making these changes late last night, not sure what happened there. I've re-included the `--require-hashes` option in the `build-homebrew` script now.